packages/api/src/filestorage/file/services/googledrive/index.ts

```
@@ -1,32 +1,180 @@
+      drive.files.list({
+        q: query,
+        fields:
+          'files(id, name, mimeType, modifiedTime, size, parents, webViewLink)',
+        pageSize: BATCH_SIZE,
+        pageToken: pageToken,
+        orderBy: 'modifiedTime',
+      }),
+    );
+
+    const files: GoogleDriveFileOutput[] = response.data.files.map((file) => ({
+      id: file.id!,
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)